File: ios/theme/RichTextTheme.m
#import "RichTextTheme.h"

@implementation RichTextTheme

+ (instancetype)defaultTheme {
    RichTextTheme *theme = [RichTextTheme new];
    theme.baseFont = [UIFont systemFontOfSize:16];  
    theme.textColor = [UIColor blackColor];
    theme.headerConfig = [HeaderConfig defaultConfig];
    return theme;
}

@end

File: android/src/main/java/com/richtext/theme/RichTextTheme.kt
package com.richtext.theme

import android.graphics.Typeface
import android.graphics.Color

data class RichTextTheme(
    val baseFont: Typeface = Typeface.DEFAULT,
    val textColor: Int = Color.BLACK,
    val headerConfig: HeaderConfig = HeaderConfig.defaultConfig()
) {
    companion object {
        fun defaultTheme(): RichTextTheme = RichTextTheme()
    }
}
